spec/lib/wisper/broadcasters/logger_broadcaster_spec.rb

```diff
@@ -27,38 +27,41 @@ module Broadcasters
         describe '#broadcast' do
           context 'without arguments' do
             let(:args) { [] }
+            let(:kwargs) { {} }
 
             it 'logs published event' do
-              expect(logger).to receive(:info).with('[WISPER] Publisher#1 published thing_created to Listener#2 with no arguments')
-              subject.broadcast(listener, publisher, event, args)
+              expect(logger).to receive(:info).with('[WISPER] Publisher#1 published thing_created to Listener#2 with no arguments and no keyword arguments')
+              subject.broadcast(listener, publisher, event, *args, **kwargs)
             end
 
             it 'delegates broadcast to a given broadcaster' do
-              expect(broadcaster).to receive(:broadcast).with(listener, publisher, event, args)
-              subject.broadcast(listener, publisher, event, args)
+              expect(broadcaster).to receive(:broadcast).with(listener, publisher, event, *args, **kwargs)
+              subject.broadcast(listener, publisher, event, *args, **kwargs)
             end
           end
 
           context 'with arguments' do
             let(:args) { [arg_double(id: 3), arg_double(id: 4)] }
+            let(:kwargs) { {x: :y} }
 
             it 'logs published event and arguments' do
-              expect(logger).to receive(:info).with('[WISPER] Publisher#1 published thing_created to Listener#2 with Argument#3, Argument#4')
-              subject.broadcast(listener, publisher, event, args)
+              expect(logger).to receive(:info).with("[WISPER] Publisher#1 published thing_created to Listener#2 with Argument#3, Argument#4 and keyword arguments #{kwargs.inspect}")
```

Comment:
Probably best for your expectation to be hard-coded. That'll match the `Publisher#1` bits in the string too. So something like this (exact output not checked).

```suggestion
              expect(logger).to receive(:info).with("[WISPER] Publisher#1 published thing_created to Listener#2 with Argument#3, Argument#4 and keyword arguments {x: :y}")
```